test/python/sdpa/frost/test_sdpa_fp8_sm107.py

```
@@ -50,3 +50,112 @@ def test_sm100_module_unchanged():
+        pytest.skip('softmax_precision="fp16" serves cc10.7 only')
+
+    from cudnn.sdpa.fwd.api_dsl import SdpaFwdDslSm100
+
+    torch.manual_seed(0)
+    B, H, S, D = 2, 4, 512, 128
+    dev = "cuda"
+    Q8, K8, V8 = ((torch.randn(B, H, S, D, device=dev) * 0.5).to(torch.float8_e4m3fn) for _ in range(3))
+    lse = torch.empty(B, H, S, device=dev, dtype=torch.float32)
+    outs = {}
+    for precision in ("fp32", "fp16"):
+        O = torch.empty(B, H, S, D, device=dev, dtype=torch.bfloat16)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)